WordPress/Sniffs/Functions/FunctionRestrictionsSniff.php

```diff
@@ -13,8 +13,8 @@ class WordPress_Sniffs_Functions_FunctionRestrictionsSniff implements PHP_CodeSn
 	 * Exclude groups
```

Comment:
This sniff file is not actually a sniff but a template/abstract class to be extended.

Most of the time, your child class will only need a custom implementation of the `getGroups()` method and can defer to this parent for the rest:
````php
class WordPress_Sniffs_Theme_DeregisterScriptSniff extends WordPress_Sniffs_Functions_FunctionRestrictionsSniff {
	public function getGroups() {
		return array(
			// The functions you want to check for and the associated error messages.
		);
	}
}
```

Changing the original `WordPress/Sniffs/Functions/FunctionRestrictionsSniff.php
` file would impact other sniffs which already extend this class, so is not a viable way to implement this.